ambry-api/src/main/java/com/github/ambry/config/StoreConfig.java

```diff
@@ -680,6 +680,12 @@ public class StoreConfig {
   @Config("store.reshuffle.disks.on.reorder")
   @Default("false")
   public final boolean storeReshuffleDisksOnReorder;
+
+  public static final String FILECOPY_FEATURE_ENABLED = "file.copy.feature.enabled";
```

Comment:
Please use this from server config: https://github.com/linkedin/ambry/pull/2966/files#diff-301f6df5395b68a10502c0c45fa96d90cb052793a96c1569c6890b07185f5aeb

Reply:
Incorporated.